### ISSUE TITLE
New switch back arrows overlap surveys/questionnaires

### ISSUE THREAD

**Issue by tooomm** [openHPI/xikolo-ios#876]
**Describe the bug**
The newly added switch buttons (https://github.com/openHPI/xikolo-ios/pull/875) are nice. They work well for e.g. videos, for tests and feedback forms they overlap with the text though.

**To Reproduce**
1. Open any course element
2. New arrows to swap back and forth between elements are shown (only works on iPad and in landscape/horizontal device orientation)
3. Video content looks good
4. Questionnaires look bad

**Expected behavior**
Move such content a bit more to the centre and do not make the new arrow overlay the text.

**Screenshots**
![ipadmini](https://user-images.githubusercontent.com/9874850/139532954-aacdd906-00af-4fb3-80b9-f2b5b3a06caa.png)

**Device (please complete the following information):**
 - Type: iPad Mini 4
 - OS: iOS 14.8.1
 - App Version: 2.12.2
 - Build number: 1984

**Comment by mathebox** [comment]
Thanks for reporting this issue in such detail! We will take care of it as soon as possible. 
Some background information: Quizzes currently are embedded as web views. So we need to fix this either in the web or need a workaround in the app.

**Comment by tooomm** [comment]
Thanks for the quick fix in 2.12.3 👍

**Comment by tooomm** [comment]
I noticed another issue after #878. It's still not optimal.
On the left side there is some extra spacing, while on the right, the button is still a little bit overlapping with the info box.

Screenshot from v2.12.3
![quizz](https://user-images.githubusercontent.com/9874850/151168957-b5c6eb02-f497-4268-9790-c529cd67c1e7.PNG)
